<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yeelink API调试工具 v1.1 by wendal.net</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>串口</string>
    </property>
    <widget class="QComboBox" name="ui_text_com_bitrate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="ui_text_com_checkbit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ODD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NDD</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_stop_read">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止读取</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_text_com_number">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_text_com_databit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_text_com_stopbit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_start_read">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始读取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>380</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>API调试服务器</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>标准服务</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>仅显示</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="ui_spin_mock_port">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>80</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>类型</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mock_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mock_stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>351</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Yeelink信息</string>
    </property>
    <widget class="QLineEdit" name="ui_text_uapikey">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Your-ApiKey</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_save_config">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存配置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_start_mqtt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MQTT启动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_get_sensors">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>获取传感器列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_stop_mqtt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MQTT关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_check_api">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>检查密钥</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>传感器列表</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>设备id</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ui_table_sensors">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>711</width>
       <height>201</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>传感器id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>传感器名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最新值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数据前缀</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数据读取前缀</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最后更新时间</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>U-ApiKey</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_help">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>帮助</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_combo_devid">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_api_test">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>API测试</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>851</width>
      <height>451</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>调试信息</string>
    </property>
    <widget class="QTextEdit" name="ui_debug_console">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>801</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="ui_checkbox_api_resp">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>API响应</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ui_checkbox_com_data">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>串口数据</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ui_checkbox_api_trace">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>详尽API输入输出</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ui_checkbox_api_send">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>API发送</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ui_checkbox_self_trace">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>软件信息</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_clear_debug">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空调试信息</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
